package/cloudshell/iac/terraform/tagging/tag_terraform_resources.py

```diff
@@ -625,33 +621,37 @@ def _get_untaggable_resources_types_from_plan_output(text: str) -> List[str]:
                                                         RegexHelper.UNSUPPORTED_TAGS_OR_LABELS_PATTERN_2])
     return untaggable_resources
 
-#modified
-def start_tagging_terraform_resources(main_dir_path: str, logger, tags_dict: dict, inputs_dict: dict = dict(), terraform_version: str = ""):
+
+# modified
+def start_tagging_terraform_resources(main_dir_path: str, logger, tags_dict: dict, inputs_dict: dict = None,
+                                      terraform_version: str = ""):
     if not os.path.exists(main_dir_path):
         raise TerraformAutoTagsError(f"Path {main_dir_path} does not exist")
     tfs_folder_path = main_dir_path
-    # log_path = Constants.get_override_log_path(main_dir_path)
-    # torque_tags_file_path = Constants.get_torque_tags_path(main_dir_path)
+    # log_path = Constants.get_override_log_path(settings.tf_module_dir_path)
```

Comment:
why not delete it instead of changing commented out line?

Reply:
removed commented out code